build-system/tasks/storybook-react/index.js

```diff
@@ -0,0 +1,36 @@
+('use strict');
+
+const argv = require('minimist')(process.argv.slice(2));
+const {build: buildBinary} = require('../build');
+const {buildEnv, launchEnv} = require('../storybook');
+const {createCtrlcHandler} = require('../../common/ctrlcHandler');
+const {defaultTask: runAmpDevBuildServer} = require('../default-task');
+/**
+ * @return {Promise<void>}
+ */
+async function storybookReact() {
+  // React dist needs to be comparied before running
+  await buildBinary();
+  const {build = false, 'storybook_env': env = 'amp,preact'} = argv;
+  const envs = env.split(',');
+  if (!build && envs.includes('amp')) {
+    await runAmpDevBuildServer();
+  }
+  if (!build) {
+    createCtrlcHandler('storybook');
+  }
+  envs.map(build ? buildEnv : launchEnv);
+}
+
+module.exports = {
+  'storybook-react': storybookReact,
+};
+
+storybookReact.description =
+  'Set up isolated development and testing for React Bento components';
+
+storybookReact.flags = {
+  'build': 'Build a static web application (see https://storybook.js.org/docs)',
+  'storybook_env': 'Environment(s) to run Storybook for react',
```

Comment:
Consider adding an example of allowed values (namely `amp` and `preact`, along with an explanation that the expected format is comma-delimited. Appending (`default: "amp,preact"`) might suffice

Reply:
This command is pretty much only going to have react for right now.

Reply:
Yep, this file is going away also.